pyopensprinkler/__init__.py

```diff
@@ -67,6 +68,18 @@ def __init__(self, url, password, opts=None):
         self._programs = {}
         self._stations = {}
         self._state = None
+        self._use_ja_endpoint = os.environ.get("PYOPENSPRINKLER_USE_JA_ENDPOINT", None)
```

Comment:
I think the option should be flipped the other way to `PYOPENSPRINKLER_NOT_USE_JA_ENDPOINT` or `PYOPENSPRINKLER_SKIP_JA_ENDPOINT` cause the current name makes it seems like the default is to not use the /ja endpoint which is not the case. 

Maybe also better to be `PYOPENSPRINKLER_SKIP_GROUPED_ENDPOINT`  or `PYOPENSPRINKLER_SKIP_ALL_ENDPOINT`

Reply:
Went with the last `PYOPENSPRINKLER_SKIP_ALL_ENDPOINT`.